patches/titleScreen.py

```diff
@@ -19,6 +19,30 @@ def _encode(s):
             result.append(CHAR_MAP.get(char, 0x7E))
     return result
 
+def _decode(byte):
+    if byte < 26:
+        return chr(byte + ord("A"))
+    elif 26 <= byte < 51:
+        return chr(byte - 26 + ord("a"))
+    elif 0x70 <= byte < 0x80:
+        return chr(byte - 0x70 + ord("0"))
+    else:
+        for k, v in CHAR_MAP.items():
+            if byte == v:
+                return k
+
+        return "?"
+
+def readSeed(rom):
+    be = BackgroundEditor(rom, 3)
```

Comment:
I don't like reading it from the "background" data, which we could change at any moment and thus breaking this. We could just store it somewhere else, somewhere in bank 3F for example (after the code, just before the graphics)

Reply:
Oh, yeah, I meant to mention this in the main PR description - I didn't feel good about this, but I also didn't know enough about how and where data is stored in the ROM to be confident making changes myself.  Feel free to change that if you'd like, or I can use it as a learning experience and take a stab at it.

Reply:
Well, banks 3E and 3F are empty in the default ROM, so I full control in those 2. I use bank 3E for a lot of code expansion and extra item tables. 3F is only used during the start of the ROM and loads graphics in the GBC VRAM, and nothing else.

In bank 3F there is some code at the start, and graphics from 0x3000 to 0x4000 (banks have a size of 0x4000)
So, placing the seed at 3F:2F00 wouldn't be an issue at all.

Reply:
Gotcha, cool.  I think I'll have time to do that tonight (EDT), thanks!

Reply:
Okay, it looks like it's storing the seed in 3F:2F00 now.